docs/components/GestaltLogo.js

```diff
@@ -0,0 +1,33 @@
+// @flow strict
+import { type Node } from 'react';
+
+type Props = {|
+  height?: number,
+  width?: number,
+|};
+
+export default function GestaltLogo({ height, width }: Props): Node {
+  return (
+    <svg
+      width={width ?? 500}
```

Comment:
@dangerismycat should we add a `aria-label` to make the logo icon more accessible? Similar to how we do this with `Icon`:
https://github.com/pinterest/gestalt/blob/2482e777d9938fae50f5e01fc36e57a3cf10c416/packages/gestalt/src/Icon.js#L126

If the answer is no, should we explicitly set `aria-hidden` to `true` on the `svg`?

Reply:
Yes, good catch! Updating now